src/components/toast/themes/dark/toast.bootstrap.scss

```diff
@@ -0,0 +1,9 @@
+@use 'sass:map';
+@use 'styles/utilities' as *;
+@use 'themes' as *;
+
+$theme: $bootstrap;
+
+:host {
+    @include css-vars-from-theme($theme, 'ig-toast');
+}
```

Comment:
I don't think this will work. The `:host` selector in the dark stylesheets will only work after [this PR](https://github.com/IgniteUI/igniteui-webcomponents/pull/927/) is merged.